main/lib/core/src/StandardPlane.cc

```diff
@@ -315,8 +315,6 @@ namespace eudaq{
   }
 
   void StandardPlane::SetupResult() const {
-    if (m_result_pix)
```

Comment:
not sure why you have removed this - it saves quite some computation time not to run this all the time.

Reply:
Added back